classes/data-stores/ActionScheduler_wpPostStore_PostStatusRegistrar.php

```diff
@@ -5,6 +5,10 @@
  * @codeCoverageIgnore
  */
 class ActionScheduler_wpPostStore_PostStatusRegistrar {
+
+	/**
+	 * Registrar.
```

Comment:
Bit of a nit, doesn't matter too much, but why *registrar* vs *register* (the latter matches the method name)?

Reply:
Matches the class name.